_docs/deployment/passenger.md

```diff
@@ -1,77 +0,0 @@
----
-layout: documentation
-title: Passenger
-category: Deployment
-order: 4.4
----
-
-# Passenger
```

Comment:
Should we redirect this page to [`/docs/guides/reverse-proxies`](https://deploy-preview-84--thelounge.netlify.com/docs/guides/reverse-proxies)?
I didn't do it here because there is no Passenger category there (and I'm not planning to add one myself), but I'm wondering if that's of any value. I doubt it, but [_cool URIs don't change_](https://www.w3.org/Provider/Style/URI) so you never know.